dist/app.js

```diff
@@ -0,0 +1 @@
ch((function(e){var n=t===r?o.c[e]:o.b[e],i=document.createElement("div");i.setAttribute("id",e),i.classList.add("key"),a.append(i),c(e,n,i,a)}))})),n&&n.forEach((function(e){var t=document.querySelector("#".concat(e.getAttribute("id")));null!==t&&t.classList.add("on-key-press")}))}var u=!1;function l(){return document.querySelector(".wrapper")}function d(){return document.querySelector("textarea")}function p(e){if(!/^(F[0-9])|(Escape)$/.test(e.code)){var t=l(),n=d(),a=e.code||e.currentTarget.getAttribute("id"),i=t.querySelector("#".concat(a));i&&!i.classList.contains("on-key-press")&&i.classList.add("on-key-press");var c=t.querySelector("#".concat(a," span")),f=t.querySelector("#".concat(a," .upper-value")),y=null!==c?c.innerText.toLowerCase():"",h=null!==f?f.innerText:y.toUpperCase();o.a.find((function(e){return e===a}))?(!function(e,t){if(e.altKey&&e.shiftKey){var n=(localStorage.getItem("currentLanguage")||r)===r?"en":r,o=document.querySelector(".keyboard"),a=document.querySelectorAll(".on-key-press");return o.innerHTML="",s(o,n,a),void localStorage.setItem("currentLanguage",n)}if("AltLeft"!==t&&"AltRight"!==t&&"ShiftLeft"!==t&&"ShiftRight"!==t&&"CtrlLeft"!==t&&"CtrlRight"!==t)if("CapsLock"!==t){var i=d(),c="";"Enter"===t&&(c+="\n"),"Tab"===t&&(c+="\t"),"Space"===t&&(c+=" ");var l=i.selectionEnd-i.selectionStart;"Backspace"===t&&0===l&&(i.selectionStart-=1),"Delete"===t&&0===l&&(i.selectionEnd+=1);var p=i.value.split(""),f=i.selectionStart;p.splice(i.selectionStart,i.selectionEnd-i.selectionStart,c),i.value=p.join(""),i.selectionStart="Delete"===t||"Backspace"===t?f:f+1,i.selectionEnd=i.selectionStart}else u=!u}(e,a),e.altKey&&e.shiftKey&&document.querySelectorAll(".key").forEach((function(e){return e.addEventListener("mousedown",p)}))):n.value+=e.shiftKey||u?h:y,e.preventDefault()}}function f(e){if(!/^(F[0-9])|(Escape)$/.test(e.code)){var t=l(),n=d();if(e.code){if("CapsLock"!==e.code||!u){var r=t.querySelector("#".concat(e.code));null!==r&&r.classList.remove("on-key-press")}}else document.body.querySelectorAll(".on-key-press").forEach((function(e){"CapsLock"===e.getAttribute("id")&&u||e.classList.remove("on-key-press")})),e.preventDefault(),n.focus()}}n(1);document.addEventListener("DOMContentLoaded",(function(){var e=localStorage.getItem("currentLanguage")||r,t=function(){var e=document.createElement("div"),t=document.createElement("textarea"),n=document.createElement("div"),r=document.createElement("p"),o=document.createElement("p");return e.classList.add("wrapper"),n.classList.add("keyboard"),r.classList.add("info"),o.classList.add("info"),r.innerText="Сделано в OS Windows 10",o.innerText="Комбинация клавиш Shift + Alt переключает язык",e.append(t,n),e.append(r),e.append(o),document.body.append(e),t.focus(),n}();s(t,e),document.body.addEventListener("keydown",p),document.body.addEventListener("keyup",f),t.querySelectorAll(".key").forEach((function(e){return e.addEventListener("mousedown",p)})),document.addEventListener("mouseup",f)}))}]);
```

Comment:
dist folder is autogenerated. 
It shouldn't be committed according to the rules: https://docs.rs.school/#/stage2?id=pull-request-%d0%bd%d0%b5-%d0%b4%d0%be%d0%bb%d0%b6%d0%b5%d0%bd-%d1%81%d0%be%d0%b4%d0%b5%d1%80%d0%b6%d0%b0%d1%82%d1%8c

Add index.html file to the project root folder as it's one of the requirements.

Reply:
Ok. Thank you for review. I fixed problems with index,html file and autogenerated dist folder